src/game/gameflow.c

```diff
@@ -201,6 +201,30 @@ static int8_t S_LoadScriptMeta(struct json_object_s *obj)
             json_array_get_number_double(tmp_arr, 2, GF.water_color.b);
     }
 
+    if (json_object_get_value(obj, "draw_distance_fade")) {
+        double value = json_object_get_number_double(
+            obj, "draw_distance_fade", JSON_INVALID_NUMBER);
+        if (value == JSON_INVALID_NUMBER) {
+            LOG_ERROR("'draw_distance_fade' must be an integer");
```

Comment:
Incorrect error message, we take doubles now

Reply:
Fixed